Lib/test/test_functools.py

```diff
@@ -233,6 +233,12 @@ def test_placeholders(self):
         actual_args, actual_kwds = p('x', 'y')
         self.assertEqual(actual_args, ('x', 0, 'y', 1))
         self.assertEqual(actual_kwds, {})
+        # Checks via `is` and not `eq`
+        # thus unittest.mock.ANY isn't treated as Placeholder
+        p = self.partial(capture, unittest.mock.ANY)
+        actual_args, actual_kwds = p()
+        self.assertEqual(actual_args, (unittest.mock.ANY,))
```

Comment:
This test is not so useful, as it will pass for any 1-tuple. You can add `self.assertIs(actual_args[0], unittest.mock.ANY)` if you want to make the test more specific. Not that it matters much here, the constructor would raise TypeError if it used `==`.